docs/src/modules/components/DemoToolbar.js

```diff
@@ -520,13 +522,32 @@ export default function DemoToolbar(props) {
               </DemoTooltip>
             </React.Fragment>
           )}
+          {enableOpenInNewTab && (
+            <DemoTooltip title={t('openInNewTab')} placement="bottom">
+              <IconButton
+                data-ga-event-category="demo"
+                data-ga-event-label={demo.gaLabel}
+                data-ga-event-action="openInNewTab"
+                onClick={() => {
+                  const url = new URL(window.location.href);
+                  url.hash = '';
+                  url.searchParams.set('scopedDemo', demoOptions.demo);
+                  window.open(url.toString(), '_blank');
+                }}
+                {...getControlProps(6)}
+                sx={{ borderRadius: 1 }}
+              >
+                <OpenInNewIcon />
+              </IconButton>
+            </DemoTooltip>
+          )}
```

Comment:
<img width="268" alt="image" src="https://github.com/user-attachments/assets/7e2b8724-8e0e-4cf8-b2ed-decc9c1b724c" />

I think it would be better to use label "Open standalone demo".

When I first see "Open in new tab", I have no clue on what to expect and not sure why should I open a new tab.